internal/transport/http2_client.go

```diff
@@ -244,7 +246,7 @@ func newHTTP2Client(connectCtx, ctx context.Context, addr resolver.Address, opts
 		if err := connectCtx.Err(); err != nil {
 			// connectCtx expired before exiting the function.  Hard close the connection.
 			if logger.V(logLevel) {
-				logger.Infof("newClientTransport: aborting due to connectCtx: %v", err)
+				logger.Infof("Aborting due to expiry of connect deadline: %v", err)
```

Comment:
Optional nit: prefer "Aborting due to connect deadline expiring: %v"

Reply:
Done.